include/ofi_mr.h

```diff
@@ -297,7 +297,7 @@ struct ofi_mr_cache {
 
 	struct ofi_mr_storage		storage;
 	struct dlist_entry		lru_list;
-	struct dlist_entry		flush_list;
+	struct dlist_entry		dead_region_list;
```

Comment:
nit: For future, rename should really be in a separate commit, but I'm fine leaving this as-is.